zerver/tests/test_events.py

```diff
@@ -2751,8 +2754,7 @@ def test_do_delete_message_stream(self) -> None:
     def test_do_delete_message_personal(self) -> None:
         schema_checker = self.check_events_dict([
             ('type', equals('delete_message')),
-            ('message_id', check_int),
-            ('sender', check_string),
+            ('message_ids', check_list(check_int, 1)),
```

Comment:
Can you add a test covering the legacy format too, by passing False for `bulk_message_deletion`?